doc/source/whatsnew/v0.24.0.txt

```diff
@@ -10,6 +10,45 @@ New features
 
 .. _whatsnew_0240.enhancements.other:
 
+.. _whatsnew_0240.enhancements.extension_array_operators
+
+``ExtensionArray`` operator support
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A ``Series`` based on ``ExtensionArray`` now supports arithmetic and comparison
+operators.  There are two approaches for providing operator support for an ExtensionArray:
+
+1. Define each of the operators on your ExtensionArray subclass. 
+2. Use an operator implementation from pandas that depends on operators that are already defined
+   on the underlying elements (scalars) of the ExtensionArray.
+
+To use the first approach where you define your own implementation of the operators,
+you define each operator such as `__add__`, __le__`, etc. on your ExtensionArray
+subclass.
+
+For the second approach, which is appropriate if your ExtensionArray contains 
+elements that already have the operators
+defined on a per-element basis, pandas provides a mixin,
+:class:`ExtensionScalarOpsMixin` that you can use that can
```

Comment:
"that you can use that can define" -> "that you can use to define"

Reply:
fixed